content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/find-out-runtime-you-use.md

```diff
@@ -0,0 +1,44 @@
+---
+title: ノードで使用されているコンテナランタイムの確認
+content_type: task
+weight: 10
+---
+
+<!-- overview -->
+
+このページでは、クラスター内のノードが使用している[コンテナランタイム](/docs/setup/production-environment/container-runtimes/)を確認する手順を概説しています。
+クラスターの実行方法によっては、ノード用のコンテナランタイムが事前に設定されている場合と、設定する必要がある場合があります。
```

Comment:
[Original text](https://github.com/kubernetes/website/blame/main/content/en/docs/tasks/administer-cluster/migrating-from-dockershim/find-out-runtime-you-use.md#L13) has empty row here.
Could you add empty row here?
```suggestion

クラスターの実行方法によっては、ノード用のコンテナランタイムが事前に設定されている場合と、設定する必要がある場合があります。
```